server/src/main/java/org/jboss/as/server/logging/ServerLogger.java

```diff
@@ -1273,6 +1273,10 @@ OperationFailedException illegalCombinationOfHttpManagementInterfaceConfiguratio
     @Message(id = 265, value = "Invalid value '%s' for system property '%s' -- value must be a non-negative integer")
     void invalidPoolCoreSize(String val, String configSysProp);
 
+    @LogMessage(level = WARN)
+    @Message(id = 266, value = "Server home is set to '%s', but server real home is '%s', This might cause problems.")
```

Comment:
This error is a bit vague, would it be of any value to mention an example of what it might cause?